app/api/chat/route.ts

```
@@ -62,6 +71,8 @@ export const POST = withWidgetAuth(async ({ request }, { session, mailbox }) =>
   const userEmail = session.isAnonymous ? null : session.email || null;
   const attachments = message.experimental_attachments ?? [];
 
+  await cacheClientTools(tools, customerSpecificTools ? conversation.emailFrom ?? userEmail ?? null : null);
```

ESLint

[error] Replace `conversation.emailFrom·??·userEmail·??·null` with `(conversation.emailFrom·??·userEmail·??·null)`

(prettier/prettier)